great_expectations/checkpoint/actions.py

```diff
@@ -606,6 +606,41 @@ def _validate_renderer(cls, renderer: dict | OpsgenieRenderer) -> OpsgenieRender
             renderer = _renderer
         return renderer
 
+    @override
+    def v1_run(self, checkpoint_result: CheckpointResult) -> dict:
```

Comment:
Lots of overlap with `_run`. I might DRY this up, but if we can remove `_run` in the near term, I'm less stressed about it.

Reply:
Legacy `_run` will be removed shortly!